content/docs/installation/features.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Features
+description: cert-manager feature policy
+---
```

Comment:
nitpick: I don't feel like I've got a great suggestion here, but it feels like "features" doesn't quite match what this page describes.  Don't get me wrong - i love the idea of adding this page.

Maybe something like "Unstable Features", "Feature Flags" or something? I think as a user if I see "features" in the sidebar I might expect like a vendor pitch of what the product can do, and I'd skip it. If I saw something like "feature flags" I'd think "oh, more things to try!" and check it out.

What do you think?

Reply:
`experimental`, `evolving`, `under development`